pkg/metrics/metrics.go

```diff
@@ -37,8 +37,10 @@ var (
 var (
 	// UserAccountGaugeVec reflects the current number of master user records in the system, with a label to partition per member cluster
 	UserAccountGaugeVec *prometheus.GaugeVec
-	// UsersPerActivationGaugeVec reflects the number of users labelled with on their current number of activations
+	// UsersPerActivationGaugeVec reflects the number of users labelled with on their current number of activations (DEPRECATED)
 	UsersPerActivationGaugeVec *prometheus.GaugeVec
+	// UsersPerActivationGaugeVec reflects the number of users labelled with on their current number of activations and email address domain
+	UserSignupsPerActivationAndDomainGaugeVec *prometheus.GaugeVec
```

Comment:
copy-paste :ghost: ?
```suggestion
	// UserSignupsPerActivationAndDomainGaugeVec reflects the number of users labelled with on their current number of activations and email address domain
	UserSignupsPerActivationAndDomainGaugeVec *prometheus.GaugeVec
```